features/fields.feature

```diff
@@ -15,6 +15,8 @@ Functionaliteit: Aanpasbare representatie met de fields parameter
   Wanneer de fields parameter is meegegeven in het request, worden alleen de attributen van de resource teruggegeven die zijn genoemd in de fields parameter, een waarde hebben en waarvoor de afnemer is geautoriseerd.
   Dus wanneer de gebruiker vraagt om gegevens waarvoor zij of hij niet geautoriseerd is, wordt er minder teruggegeven dan er gevraagd is in de fields-parameter.
 
+  Wanneer het endpoint een collectie van resources teruggeeft, worden in fields de properties van de resource teruggegeven. De HAL elementen van de collectie wordt hierbij niet opgenomen.
```

Comment:
Wat wordt er bedoeld met **De HAL elementen van de collectie wordt hierbij niet opgenomen**?

Reply:
Als ik dit zo lees begrijp ik dat ook niet, ondanks dat ik weet wat er wordt bedoeld.

Reply:
Is het zo duidelijker?

`Wanneer het endpoint een collectie van resources teruggeeft, worden in fields de properties van de resource opgenomen. Bovenliggende berichtelementen die de collectie definieren worden hierbij niet opgenomen. Om bijvoorbeeld de straat en huisnummer te vragen in een collectie van adressen gaat met fiels=straat,huisnummer. Dus niet met _embedded.adressen.straat,_embedded.adressen.huisnummer.`

Reply:
Wel duidelijker, maar 1 typo. fields ipv fiels.
Melvin is op vakantie. Ik denk dat we niet op zijn reactie moeten wachten met mergen...